### PR TITLE
Add spice.ai cloud platform quickstart

### PR DESCRIPTION
**Pull request by digadeesh**

### CODE REVIEW

File: spiceai/README.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+```bash
+spice init spiceai-demo
+```
+
+```bash
+cd spiceai-demo
+spice run
+```
+
+**Step 3.** Configure the dataset to connect to Spice.ai:
```
Comment (Jeadie) [inline spiceai/README.md:32]: May be worth saying
1. In a new console
2. Specifically in `./spiceai-demo`

File: spiceai/README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+Locally accelerate (y/n)? n
+```
+
+The CLI will confirm the dataset has been configured with the following output:
+
+```bash
+Saved datasets/eth_recent_blocks/dataset.yaml
+```
+
+Thhe content of dataset.yaml is the following:
```
Comment (Jeadie) [inline spiceai/README.md]:
```suggestion
The content of dataset.yaml is the following:
```

Comment (Jeadie) [reply spiceai/README.md]: Also might be worth having the command `cat datasets/eth_recent_blocks/dataset.yaml` so a user can check it.

File: spiceai/README.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+Enter the description of the dataset:
+
+```bash
+description: ethereum recent block data
+```
+
+Specify the location of the dataset:
+
+```bash
+from: spiceai/eth.recent_blocks
```
Comment (Jeadie) [inline spiceai/README.md]:
```suggestion
from: spice.ai/eth.recent_blocks
```

File: spiceai/README.md (hunk 1 above)
Comment (digadeesh) [reply spiceai/README.md:32]: will do.

File: spiceai/README.md (hunk 2 above)
Comment (digadeesh) [reply spiceai/README.md]: added.  Thanks for the feedback.

File: spiceai/README.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+**Limitations**
+When accelerating Spice.ai datasets, only the following datasets support an append refresh mode:
+
+* eth.recent_blocks
+* eth.recent_transactions
+* eth.recent_traces
+
+All other datasets must be configured with `refresh_mode: full`.
+
+ßß
```
Comment (Jeadie) [inline spiceai/README.md]: ?

**Review by Jeadie — DISMISSED** [review]

**Review by sgrebnov — APPROVED** [review]

**Pull request merged**